oak-lucene/src/main/java/org/apache/jackrabbit/oak/plugins/index/lucene/property/PropertyIndexCleaner.java

```diff
@@ -97,6 +100,35 @@ public void run() {
         }
     }
 
+    /**
+     * Perform some cleanup.
+     *
+     * @param paths the list of paths (comma separated)
+     * @param batchSize the bach size
+     * @param sleepPerBatch the number of milliseconds to sleep per batch
+     * @param maxRemoveCount the maximum number of nodes to remove per path
+     * @return the number of nodes removed
+     */
+    public int performCleanup(String paths, int batchSize, int sleepPerBatch, int maxRemoveCount) throws CommitFailedException {
+        ArrayList<String> list = new ArrayList<>(Arrays.asList(paths.split(",")));
```

Comment:
Could this be simplified with the following?
```suggestion
        String[] list = paths.split(",");
```